zerver/views/webhooks/hellosign.py

```diff
@@ -0,0 +1,60 @@
+from __future__ import absolute_import
+from django.utils.translation import ugettext as _
+from zerver.lib.actions import check_send_message
+from zerver.lib.response import json_success, json_error
+from zerver.decorator import REQ, has_request_variables, api_key_only_webhook_view
+
+from zerver.models import Client, UserProfile
+
+from django.http import HttpRequest, HttpResponse
+from six import text_type
+from six.moves import range
+from typing import Dict, Any, Optional, Tuple, Union
+
+def format_body(signatories, model_payload):
+    # type: (List[Dict[str, Any]], Dict[str, Any]) -> Tuple[Any, str]
+    def separator(i, len_sig):
+        # type: (int, int) -> None
+        if i + 1 == len(signatories):
+            result.append('.')
+        elif i + 2 == len(signatories):
+            result.append(' and')
+        elif i + 3 != len(signatories):
+            result.append(',')
+
+    result = ["The {}".format(model_payload['contract_title'])] # type: Any
```

Comment:
As commented on chat - using `Any` here is appropriate as per mypy features and docs. Changing a variable type in the same function is not supported - http://mypy.readthedocs.io/en/latest/common_issues.html#redefinitions-with-incompatible-types. You'd have to use multiple variables to assure detailed typing here.